sqldelight-compiler/src/test/kotlin/com/squareup/sqldelight/core/tables/InterfaceGeneration.kt

```diff
@@ -149,18 +150,50 @@ class InterfaceGeneration {
       |        override val doubleValue: kotlin.Double,
       |        override val blobValue: kotlin.ByteArray
       |    ) : com.example.Test {
-      |        override fun toString(): kotlin.String = ""${'"'}
-      |        |Test.Impl [
-      |        |  intValue: ${"$"}intValue
-      |        |  intValue2: ${"$"}intValue2
-      |        |  booleanValue: ${"$"}booleanValue
-      |        |  shortValue: ${"$"}shortValue
-      |        |  longValue: ${"$"}longValue
-      |        |  floatValue: ${"$"}floatValue
-      |        |  doubleValue: ${"$"}doubleValue
-      |        |  blobValue: ${"$"}blobValue
-      |        |]
-      |        ""${'"'}.trimMargin()
+      |        override fun toString(): kotlin.String = buildString {
+      |            appendln("Test.Impl [")
+      |            appendln(${'"'}""  intValue: ${"$"}intValue""${'"'})
+      |            appendln(${'"'}""  intValue2: ${"$"}intValue2""${'"'})
+      |            appendln(${'"'}""  booleanValue: ${"$"}booleanValue""${'"'})
+      |            appendln(${'"'}""  shortValue: ${"$"}shortValue""${'"'})
+      |            appendln(${'"'}""  longValue: ${"$"}longValue""${'"'})
+      |            appendln(${'"'}""  floatValue: ${"$"}floatValue""${'"'})
+      |            appendln(${'"'}""  doubleValue: ${"$"}doubleValue""${'"'})
+      |            appendln(${'"'}""  blobValue: ${"$"}{blobValue.kotlin.collections.contentToString()}""${'"'})
```

Comment:
Looks like this output is incorrect.

Reply:
It's just not wrapped into a `FileSpec` hence the function doesn't get imported.